css/ampshared.css

```diff
@@ -539,6 +539,19 @@ amp-accordion {
   display: block !important;
 }
 
+/* heading */
+ .i-amphtml-accordion-header {
+  margin: 0;
+  cursor: pointer;
+  background-color: #efefef;
+  padding-right: 20px;
+  border: solid 1px #dfdfdf;
+}
+/* heading/content */
+.i-amphtml-accordion-content {
+  margin: 0;
+}
```

Comment:
This won't be enough because the `i-amphtml-accordion-header` and `i-amphtml-accordion-content` class names are added at runtime. This would have to be paired with an Optimizer transformer.

Reply:
Perhaps rather include this rule from v1.0 instead:

https://github.com/ampproject/amphtml/blob/92abb24cd5d2b7741aae7097fc2e2a8f43cf1a96/extensions/amp-accordion/1.0/amp-accordion.css#L41-L53

Reply:
Absolutely. Those changes should be done outside of this PR if it's approved though.

Reply:
I've modified this PR to use the approach in your second comment here. It's a nice tradeoff of supported features and less overall work for involved systems.